### PR TITLE
media-video/ffmpeg: add support for zeromq

### PR DESCRIPTION
**Pull request by iSchluff**
Enables use of the zmq and azmq filters for filtergraph messaging.

### CODE REVIEW

File: media-video/ffmpeg/ffmpeg-3.2.2.ebuild [hunk 1]
```diff
@@ -238,6 +238,7 @@ RDEPEND="
 		>=x11-libs/libXv-1.0.10[${MULTILIB_USEDEP}]
 	)
 	xcb? ( >=x11-libs/libxcb-1.4[${MULTILIB_USEDEP}] )
+	zeromq? ( >=net-libs/zeromq-4.1.6 )
```
Comment (aballier) [inline media-video/ffmpeg/ffmpeg-3.2.2.ebuild]: no multilib here ?

**Comment by aballier** [comment]
please also add it to -9999 ebuild otherwise it'll be lost on next version

File: media-video/ffmpeg/ffmpeg-3.2.2.ebuild (hunk 1 above)
Comment (iSchluff) [inline media-video/ffmpeg/ffmpeg-3.2.2.ebuild]: the zeromq-ebuild doesn't offer multilib support

Comment (aballier) [inline media-video/ffmpeg/ffmpeg-3.2.2.ebuild]: Then you need to disable it for non native ABIs like it is done for frei0r and netcdf. It is preferable to add multilib support to zmq though.

**Comment by aballier** [comment]
something seems wrong:

```
 * abi_x86_32.x86: running multilib-minimal_abi_src_configure
/var/tmp/portage/media-video/ffmpeg-3.2.2/work/ffmpeg-3.2.2/configure --prefix=/usr --libdir=/usr/lib32 --shlibdir=/usr/lib32 --docdir=/usr/share/doc/ffmpeg-3.2.2/html --mandir=/usr/share/man --enable-shared --cc=x86_64-pc-linux-gnu-gcc -m32 --cxx=x86_64-pc-linux-gnu-g++ -m32 --ar=ar --optflags=-march=skylake -mtune=skylake -O2 -pipe -ggdb --disable-static --enable-avfilter --enable-avresample --disable-stripping --enable-version3 --enable-version3 --disable-indev=oss --disable-indev=jack --disable-outdev=oss --enable-version3 --enable-version3 --enable-nonfree --enable-bzlib --disable-runtime-cpudetect --disable-debug --enable-gcrypt --enable-gnutls --enable-gmp --enable-gpl --enable-hardcoded-tables --enable-iconv --enable-lzma --enable-network --disable-openssl --enable-postproc --enable-libsmbclient --enable-ffplay --enable-sdl2 --enable-vaapi --disable-vdpau --enable-xlib --enable-libxcb --enable-libxcb-shm --enable-libxcb-xfixes --enable-zlib --enable-libcdio --enable-libiec61883 --enable-libdc1394 --enable-libcaca --enable-openal --enable-opengl --enable-libv4l2 --disable-libpulse --enable-libopencore-amrwb --enable-libopencore-amrnb --enable-libfdk-aac --enable-libopenjpeg --enable-libbluray --enable-libcelt --enable-libgme --enable-libgsm --disable-mmal --disable-libmodplug --enable-libopus --disable-libilbc --enable-librtmp --disable-libssh --enable-libschroedinger --enable-libspeex --enable-libvorbis --enable-libvpx --enable-libzvbi --enable-libbs2b --enable-chromaprint --enable-libebur128 --enable-libflite --enable-frei0r --enable-libfribidi --enable-fontconfig --disable-ladspa --enable-libass --enable-libfreetype --enable-librubberband --enable-libzmq --enable-libzimg --enable-libsoxr --enable-pthreads --enable-libvo-amrwbenc --enable-libmp3lame --enable-libkvazaar --disable-nvenc --enable-libopenh264 --enable-libsnappy --enable-libtheora --disable-libtwolame --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --disable-frei0r --disable-zeromq --disable-amd3dnow --disable-amd3dnowext --disable-fma4 --disable-xop --cpu=skylake --disable-doc --disable-htmlpages --disable-manpages
Unknown option "--disable-zeromq".
See /var/tmp/portage/media-video/ffmpeg-3.2.2/work/ffmpeg-3.2.2/configure --help for available options.
 * ERROR: media-video/ffmpeg-3.2.2::gentoo failed (configure phase):
 *   (no error message)
```

**Comment by aballier** [comment]
merged, thanks

I can't include the 3.2 part since it is now stable on some arches and I don't want to retroactively add untested features; it'll be available on 3.3

**Pull request closed without merging**